pom.xml

```diff
@@ -211,115 +211,115 @@
                 <version>3.1.8</version>
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi</groupId>
+                <groupId>com.github.cafapi.common.api</groupId>
```

Comment:
This makes me realize we made a mistake in `caf-common`.  We need to discuss this.

Reply:
Ok it was supposed to be just `com.github.cafapi.common` for all caf-common artifacts.

Reply:
Right

Reply:
Will fix them.